go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -186,6 +188,14 @@ func runOvnKube(ctx *cli.Context) error {
 		panic(err.Error())
 	}
 
+	// Set up a watch on our config file; if it changes, we should exit -
+	// (we don't have the ability to dynamically reload config changes).
+	if configFile != "" {
+		if err := watchForChanges(configFile, stopChan); err != nil {
```

Comment:
Do you know if stopChan is wired up? It will stop the informers, for sure, but will it stop the actual process?

Please test this, and I suspect you'll have to add something to line 252 to shut down when stopChan is closed.

Reply:
@squeed I am working on testing this. I found this code and tried to integrate it. I really appreciate a good close look at this.

Reply:
Added os.Exit() Tested and it works.